### PR TITLE
feat: Publish extension to open-vsx.org

### PR DESCRIPTION
**Pull request by carmenberndt**
On push to master the versions are bumped and pushed to master. The `publish-marketplace` and `publish-open-vsx` workflows are triggered after, taking what's on master publishing it to both marketplace and open-vsx.

If publishing to e.g. marketplace fails, the workflow can just be re-triggered.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -18,5 +19,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install Dependencies
         run: npm install
-      - name: Publish Extension as Patch
+      - name: Publish Extension as Patch to marketplace
         run: npm run vsce:publish
+      - name: Publish Extension as Patch to open-vsx
+        run: ovsx publish -p "$OPEN_VSX_ACCESS_TOKEN"
```
Comment (divyenduz) [inline .github/workflows/publish.yml:25]: This should be `yarn ovsx` or `node_modules/.bin/ovsx`, `node_modules` binaries are not registered globally so this won't work afaik.

**Comment by carmenberndt** [comment]
> looks great @carmenberndt! I see @divyenduz's comment addressed.
> 
> Next up after this, we were thinking to add an "insiders" package for pre-release on every merge, and then @divyenduz and I thought we could just publish stable releases locally? What do you think?

Sounds good!

File: .github/workflows/publish-marketplace.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    branches: [master]
+    types: 
+      - completed
+  workflow_dispatch
+
+env:
+  PAT: ${{ secrets.PAT }}
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRODUCTION: ${{ secrets.PRODUCTION }}
+  OPEN_VSX_ACCESS_TOKEN: ${{ secrets.OPEN_VSX_ACCESS_TOKEN }}
```
Comment (acao) [inline .github/workflows/publish-marketplace.yml:15]: i think this env variable would not be needed now it's seperated into two yaml files. same with `PAT` in the open-vsx action

Comment (carmenberndt) [reply .github/workflows/publish-marketplace.yml]: yes, good point, removed those 👍

**Pull request merged**